src/Walker.ts

```diff
@@ -118,4 +118,24 @@ export class Walker {
     await this.loadProductionDependencies();
     await this.pruneModule(this.rootModule);
   }
+
+  private getNearestModuleDirectory(tPath: string) {
+    while (!fs.statSync(tPath).isDirectory() || !fs.existsSync(path.resolve(tPath, 'package.json'))) {
+      tPath = path.dirname(tPath);
+    }
+    return tPath;
+  }
+
+  async getFsCopyIgnoreFunction() {
+    await this.loadProductionDependencies();
+    const prodPaths = this.prodPaths;
```

Comment:
@malept In case someone runs `prune` or something in the mean time.  Would allow people to use the same `pruner` object for multiple things at the same time.